seccomp_internal.go

```diff
@@ -210,7 +220,7 @@ const (
 	scmpError C.int = -1
 	// Comparison boundaries to check for architecture validity
 	archStart ScmpArch = ArchNative
-	archEnd   ScmpArch = ArchS390X
+	archEnd   ScmpArch = ArchPARISC64
```

Comment:
Another newbie question here - this feels error prone.  In C, we would often use a value at the end of the enumeration (say ARCH_LAST or something) so that the actual code doesn't need to change.  What is golang's stance on this?

Reply:
I'm not sure how most golang folks do this, but it certainly fits with how we've handled it thus far so I see no reason to hold this PR up.  However, I do agree that it seems fragile.